src/main.ts

```diff
@@ -1,15 +1,27 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from '@/app.module';
 import { ConfigService } from '@nestjs/config';
+import { NestExpressApplication } from '@nestjs/platform-express';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import * as packageJson from 'package.json';
```

Comment:
I don't like the idea of importing from a json file. This would require allowing json imports all across the app which is a bad thing IMHO. Please remove this and just duplicate the description and version for now.

Reply:
Okay